templates/taxi/car_form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  {% if object %}
+    <h1>Update a car:</h1>
+  {% else %}
+    <h1>Add a car:</h1>
+  {% endif %}
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
+
+{% endblock %}
```

Comment:
Please ensure that you are using 2 whitespaces indentation in your `.html` files.